Packs/SafeNet_Trusted_Access/IncidentTypes/incidenttype-SafeNet_Trusted_Access_Account_Lockout_Alert.json

```diff
@@ -0,0 +1,28 @@
+{
+    "autorun": true,
+    "color": "#B7E7FF",
+    "days": 0,
+    "daysR": 0,
+    "default": false,
+    "detached": false,
+    "disabled": false,
+    "extractSettings": {
+        "fieldCliNameToExtractSettings": {},
+        "mode": "Specific"
+    },
+    "hours": 0,
+    "hoursR": 0,
+    "id": "SafeNet Trusted Access Account Lockout Alert",
+    "layout": "SafeNet Trusted Access – Account Lockout Incident Layout",
+    "locked": false,
+    "name": "SafeNet Trusted Access Account Lockout Alert",
+    "onChangeRepAlg": 0,
+    "playbookId": "SafeNet Trusted Access - Terminate User SSO Sessions",
+    "readonly": false,
+    "reputationCalc": 0,
+    "system": false,
+    "version": -1,
+    "weeks": 0,
+    "weeksR": 0,
+    "fromVersion": "5.5.0"
```

Comment:
If there is no specific reason for this incident type to has fromvesion 5.5.0, please change to 6.0.0.
```suggestion
    "fromVersion": "6.0.0"
```

Reply:
Done.

![image](https://user-images.githubusercontent.com/86044314/140712082-c2a03548-75bf-45c6-82df-101a5fa72f07.png)